server/src/services/plugin.service.ts

```diff
@@ -111,12 +116,12 @@ export class PluginService extends BaseService {
   }
 
   private async loadPluginToDatabase(manifest: PluginManifestDto, basePath: string): Promise<void> {
-    const currentPlugin = await this.pluginRepository.getPluginByName(manifest.name);
-    if (currentPlugin != null && currentPlugin.version === manifest.version) {
-      this.logger.log(`Plugin ${manifest.name} is up to date (version ${manifest.version}). Skipping`);
-      return;
-    }
-
+    // const currentPlugin = await this.pluginRepository.getPluginByName(manifest.name);
```

Comment:
maybe you should just add something that uses `isDev` to skip plugin initialization in some situations, so you don't have to do stuff like this.